### PR TITLE
Spec for Task Detail Page

### PR DESCRIPTION
**Pull request by kamleshverma1**
The intent of task details page is to provide a current status for a specific task which include submitted time, submitted by, progress bar, execution steps, events, logs into tendrl for track down the specific task or job.

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk @brainfunked @nthomas-redhat @anivargi @cloudbehl Please review it.

**Comment by kamleshverma1** [comment]
Linked to https://github.com/Tendrl/specifications/issues/122

File: specs/task-details-view.adoc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+None
+
+=== Data model impact:
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+API which provide information about the message metadata for task status updates on the UI - Required
```
Comment (kamleshverma1) [inline specs/task-details-view.adoc:51]: @brainfunked @anivargi Please provide information about the message metadata for task status updates

File: specs/task-details-view.adoc [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+==== Tendrl frontend impact:
+
+   * Needs to add one more routing for task details view.
+   * Needs to be add one more menu inside navigation.
+
+==== Tendrl Backend impact:
+
+This is discussed in a different specification - 
+Required
```
Comment (kamleshverma1) [inline specs/task-details-view.adoc]: @brainfunked @anivargi Please provide me all specs link related to logging specification , i need to add that in this spec as dependency.

Comment (kamleshverma1) [reply specs/task-details-view.adoc]: resolved

File: specs/task-details-view.adoc (hunk 1 above)
Comment (brainfunked) [reply specs/task-details-view.adoc:51]: @anivargi please sync up with @GowthamShanmugam to provide the message format to this specification. The review comments at https://github.com/Tendrl/specifications/pull/94#pullrequestreview-17399508 would need to be addressed for the message format to be finalised.

File: specs/task-details-view.adoc [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+API which provide information about the message metadata for task status updates on the UI - Required
+
+
+==== Tendrl frontend impact:
+
+   * Needs to add one more routing for task details view.
```
Comment (shtripat) [inline specs/task-details-view.adoc:56]: Be specific instead of saying `more`

File: specs/task-details-view.adoc [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+* https://github.com/Tendrl/specifications/issues/122
+
+
+== Testing:
+
+=== Unit tests needs to be tested -
+
+* Should set the initialization properties correct.
+* Should set the details of task details from the view.
```
Comment (shtripat) [inline specs/task-details-view.adoc:114]: Should be able to view the task details from the view.

**Pull request closed without merging**